src/components/availability-label/helper.ts

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import { useGetAvailabilityV3 } from "../../core/fbs/fbs";
+import { useGetV1ProductsIdentifier } from "../../core/publizon/publizon";
+import { useConfig } from "../../core/utils/config";
+
+export const onlineMaterialAllowList = ["lydbog (net)"];
+
+export const isMaterialTypeOnline = (materialType: string) =>
+  onlineMaterialAllowList.includes(materialType);
+
+export const useAvailabilityData = ({
+  materialType,
+  faustIds,
+  isbn,
+  isTrue
+}: {
+  materialType: string;
+  faustIds: string[];
+  isbn: string;
+  isTrue: boolean;
+}) => {
+  const [isAvailable, setIsAvailable] = useState(false);
+  const config = useConfig();
+  const blacklistBranches = config("blacklistedAvailabilityBranchesConfig", {
+    transformer: "stringToArray"
+  });
+
+  // Forcing isAvailable to true if isTrue is true
+  useEffect(() => {
+    if (isTrue) setIsAvailable(true);
+  }, [isTrue]);
+
+  const hasIsbn = Boolean(isbn);
```

Comment:
Please reconsider this.

Instead of making isbn a string which can be interpreted as a boolean if it is empty I suggest making the isbn parameter nullable or optional.